### DISCUSSION TITLE
What is the purpose of the functions to limit conversational memory?

### DISCUSSION THREAD

**Discussion by xyzelectronic** [n3d1117/chatgpt-telegram-bot#335] [Q&A]
Default messages is 15 and/or 180 minutes to restrict token usage, but I'm a little confused, I thought those tokens were already spent, so to say.

**Comment by n3d1117** [comment]
Hey there, tokens in conversations are cumulative, which means each new message within the same conversation becomes more expensive to use. The usage could grow indefinitely, this is why these limits are in place.
